charts/pihole/templates/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       annotations:
         checksum.config.adlists: {{ include (print $.Template.BasePath "/configmap-adlists.yaml") . | sha256sum | trunc 63 }}
         checksum.config.blacklist: {{ include (print $.Template.BasePath "/configmap-blacklist.yaml") . | sha256sum | trunc 63 }}
+        checksum.config.ftl: {{ include (print $.Template.BasePath "/configmap-ftl.yaml") . | sha256sum | trunc 63 }}
```

Comment:
Without this, the pod won't restart on a change to that configmap, and the content won't reload either inside the pod because `subPath` is used and then contents are only updated on startup.